earth_enterprise/rpms/build.gradle

```diff
@@ -44,7 +44,7 @@ apply plugin: "nebula.ospackage"
 def rpmPlatformString = com.tsciences.os.Platform.rpmPlatformString
 
 def openGeeVersion =
-    new File(project.projectDir, '../src/fusion_version.txt').readLines().
+    new File('/tmp/fusion_os_install/gee_long_version.txt').readLines().
```

Comment:
This needs to use the stagedInstallDir variable (defined below) to determine the path instead of hard-coding to /tmp/fusion_os_install